content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -153,6 +159,20 @@ When reviewing, use the following as a starting point.
 
 ### Other
 
-For small issues with a PR, like typos or whitespace, prefix your comments with `nit:`.
-This lets the author know the issue is non-critical.
-
+- Watch out for [trivial edits](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits);
+  if you see a change that you think is a trivial edit, please point out that policy
+  (it's still OK to accept the change if it is genuinely an improvement).
+- Encourage authors who are making whitespace fixes to do
+  so in the first commit of their PR, and then add other changes on top of that. This
+  makes both merges and reviews easier. Watch out especially for a trivial change that
+  happens in a single commit along with a large amount of whitespace cleanup
+  (and if you see that, encourage the author to fix it).
+
+As a reviewer, if you identify small issues with a PR that aren't essential to the meaning,
+such as typos or incorrect whitespace, prefix your comments with `nit:`.
+This lets the author know that this part of your feedback is non-critical.
+
+If you are considering a pull request for approval and all the remaining feedback is
+marked as a nit, you can merge the PR anyway. In that case, it's often useful to open
+an issue about the remaining nits. Consider whether you're able to meet the requirements
+for mark that new issue as a “Good First Issue”; if you can, these are a good source.
```

Comment:
```suggestion
for mark that new issue as a [Good First Issue](https://github.com/kubernetes/community/blob/master/contributors/guide/help-wanted.md#good-first-issue); if you can, these are a good source.
```

minor enhancement :)

Reply:
Is this good enough to merge without that tweak? I'd hope that these new guidelines nudge folks towards getting things merged and away from waiting for perfection.

PS. I'd rather link to https://www.kubernetes.dev/docs/guide/help-wanted/#good-first-issue

Reply:
@sftim 
I think providing link is optional. Please feel free to update or not !
Let me mark this conversation is resolved. :)